Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/BaseClasses/MotorMachineInterface.mo

```diff
@@ -0,0 +1,130 @@
+    annotation (Placement(transformation(
+        extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-120,40}),
+        iconTransformation(
+        extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-120,40})));
+  Modelica.Blocks.Interfaces.RealInput f(final quantity="Frequency",
+    final unit="Hz")
+    "Controllable freuqency to the motor"
+    annotation (Placement(transformation(
+        extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-120,0}),
+        iconTransformation(
+        extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-120,0})));
+  Modelica.Blocks.Interfaces.RealInput omega_r(final quantity="AngularVelocity",
+    final unit="rad/s")
+    "Prescribed rotational speed of rotor"
+    annotation (Placement(transformation(
+        extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-120,-40}),
+        iconTransformation(
+        extent={{-20,-20},{20,20}},
+        rotation=0,
+        origin={-120,-40})));
+  Modelica.Blocks.Interfaces.RealOutput tau_e(final quantity="Torque",
+    final unit="N.m")
+    "Electromagenetic torque of rotor"
+    annotation (Placement(transformation(extent={{100,-20},{140,20}}),
+      iconTransformation(extent={{100,-20},{140,20}})));
+
+protected
+  Real ratio "Intermediate value used in calculation";
+
+equation
+
+  omega_s = 4*Modelica.Constants.pi*f/pole;
+  s = if noEvent(omega_s>0) then 1-omega_r/omega_s else 0;
+  ratio = X_m/(X_m+X_s);
+  tau_e = if noEvent(omega_s>0)
+  then n*R_r*s*(V_rms*ratio)^2/(omega_s*(((s*R_s*ratio^2+R_r)^2)+s^2*(X_s+X_r)^2))
+  else 0;
+
+  annotation (Icon(coordinateSystem(preserveAspectRatio=true), graphics={
+          Rectangle(
+          extent={{-100,100},{100,-100}},
+          lineColor={0,0,127},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.Solid),
+          Text(
+          extent={{-82,162},{82,116}},
+          lineColor={0,0,255},
+          fillColor={255,255,255},
+          fillPattern=FillPattern.None,
+          textString="%name")}),
+          defaultComponentName="torSpe",
+    Documentation(info="<html>
+<p>
+This block is to implement equations of the simplified model of squirrel cage type 
+induction motor. The induction motor operation is based on the principle of 
+electromagnetic induction; there are two magnetic fields occurring, one from 
+the stator and one from the rotor, the rotor field is induced by the stator 
+field. Therefore, an induction motor can be modeled as a rotating transformer. 
+</p>
+<p>
+The per-phase equivalent circuit model for an induction motor supplied by 
+balanced three-phase supply based on the transformer model is represented below. 
+</p>
+<p align=\"center\"><img src=\"modelica://Buildings/Resources/Images/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/InductionMotors/induction_motor_equivalent_circuit.png\"/> 
```

Comment:
Check the syntax, it will cause html check warning as: `<img> lacks "alt" attribute`.
see for example here:
https://github.com/lbl-srg/modelica-buildings/blob/79f20f28c229fca01a802b61a7b391bbf585109e/Buildings/Controls/OBC/CDL/Continuous/Acos.mo#L55